tests/test_planet_routes.py

```diff
@@ -0,0 +1,72 @@
+
+
+def test_get_planets(client):
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # assert
+    assert response.status_code == 200
+    assert response_body == []
+
+
+
+def test_get_single_planet(client, three_saved_planets):
+    response = client.get("/planets/1")
```

Comment:
If we returned the record list from the fixture, then we could write this line as
```py
    response = client.get(f"/planets/{three_saved_planets[0].id}")
```